ocaml/xapi/cli_operations.ml

```diff
@@ -2665,16 +2665,21 @@ let get_snapshot_uuid params =
   then List.assoc "uuid" params
   else raise (failwith "Required parameter not found: snapshot-uuid or uuid.")
 
+let get_snapshotVM_by_uuid rpc session_id snap_uuid =
+  let snap_ref = Client.VM.get_by_uuid rpc session_id snap_uuid in
+  if not (Client.VM.get_is_a_snapshot rpc session_id snap_ref) then failwith "This operation can only be done on VM snapshot.";
```

Comment:
Suggestion: `"This operation can only be performed on a VM snapshot. %s is not a VM snapshot." snap_uuid`.

Reply:
Chris, I have updated the message as you suggested, would you fine with the current wording? Do you still have other comments, or may I proceed to rebase from master and squash the comments to prepare for merge?